test/e2e/specs/wp-theme-multisite-spec.js

```diff
@@ -149,7 +149,8 @@ describe( `[${ host }] Themes: All sites (${ screenSize })`, function () {
 					return await this.siteSelector.ok();
 				} );
 
-				describe( 'Successful activation dialog', function () {
+				// Skip reason:
```

Comment:
Should we add the reason here? Or maybe remove this comment?

Reply:
Oh whoops, I meant to link the issue, I'll open another PR to fix that, thanks for catching it.

Reply:
Fixed here @fullofcaffine https://github.com/Automattic/wp-calypso/pull/50136